packages/core/src/middleware/after-response/prepare-response.ts

```diff
@@ -4,10 +4,17 @@ import type { ModifiedResponse } from '../../types'
 const prepareResponse: AfterResponseHook = async (_request, _options, response) => {
   const modifiedResponse = response as ModifiedResponse
 
-  // Clone the response before reading the body to avoid
-  // `TypeError: body used already` elsewhere
-  const clone = response.clone()
-  const content = await clone.text()
+  let content: string
+  if (_options.skipResponseCloning) {
```

Comment:
I think convention is if we are using a variable then it should no longer be `_options`. This convention comes from golang. https://stackoverflow.com/questions/27764421/what-is-underscore-comma-in-a-go-declaration

So you should rename `_options` to `options`.

Reply:
Is `_options` guaranteed to be defined? Wonder if should do something like:

```suggestion
  if (_options && _options.skipResponseCloning) {
```

Reply:
Does not seem that it can be undefined as per Typescript conventions:
![image](https://user-images.githubusercontent.com/103517471/173418391-e3641ac7-60f3-4ba9-8bb9-230cf53410ef.png)

Reply:
^ Manually changing the type to `NormalizedOptions | undefined` results in this linting rule so this check is automated by the framework already 
![image](https://user-images.githubusercontent.com/103517471/173418561-63524ff1-d816-441b-a75d-51022c60d42b.png)